### DISCUSSION TITLE
Unable to log Flow Errors

### DISCUSSION THREAD

**Discussion by crouchlizardo** [jongpie/NebulaLogger#731] [Q&A]
Hey!

I've recently added the unmanaged package to a sandbox and I am currently going through ways to see how logs are created. I seem to be missing something (something I feel may be obvious - so I apologise!)..

When debugging my flow, it says the flow entry is created - infact a transaction ID is returned (This is similar to something which a user has posted before (https://github.com/jongpie/NebulaLogger/discussions/675) However, when I took the fix steps - it still doesnt show any logs..

Screenshot of flow action
![image](https://github.com/user-attachments/assets/a96f9b93-e7fa-48ce-9b4c-5a560972654e)

Screenshot of settings - Now I imagine I have set this up COMPLETELY wrong (sorry if I have missed something obvious!)
![image](https://github.com/user-attachments/assets/4ab24446-8465-4686-a119-abb1f20e8665)

Any support would be appreciated thank you so much

**Comment by jongpie** [comment]
@crouchlizardo it looks like you have the property "(Optional) Save Method" set to `true` - that is not the correct type of value for that property, it's supposed to either not be set (recommended), or set to a text value (accepted values are `EVENT_BUS`, `QUEUEABLE`, `REST`, or `SYNCHRONOUS_DML`).

I'd suggest trying:
- Don't set "Save Method"
- Do set "Save Log", and this one _should_ be set to `true`

**Reply by crouchlizardo** [reply]
> @crouchlizardo it looks like you have the property "(Optional) Save Method" set to `true` - that is not the correct type of value for that property, it's supposed to either not be set (recommended), or set to a text value (accepted values are `EVENT_BUS`, `QUEUEABLE`, `REST`, or `SYNCHRONOUS_DML`).
> 
> I'd suggest trying:
> 
> * Don't set "Save Method"
> * Do set "Save Log", and this one _should_ be set to `true`

Hi Jon,
Thanks for taking a look at this, I have applied the changes but still not having any luck with it creating a log

Interestingly, I have added the managed package to a different sandbox and it works perfectly with the flow - I will use the managed package for now :)
